sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -665,7 +700,7 @@ public DigitalTwinsServiceVersion getServiceVersion() {
                         objectPagedResponse.getStatusCode(),
                         objectPagedResponse.getHeaders(),
                         convertedList,
-                        null,
```

Comment:
This one was odd to me. I'm assuming we didn't mean to assume that there was no continuation token